### PR TITLE
support moore threads MUSA arch

### PR DESCRIPTION
**Pull request by hanhaowen-mt**
## Motivation

support moore threads MUSA gpu

## Modification

Due to compatibility with cuda, cuda operators can be directly ported

### CODE REVIEW

**Comment by hanhaowen-mt** [comment]
https://github.com/open-mmlab/mmengine/pull/1453
mmengine have already supported MUSA

**Comment by hanhaowen-mt** [comment]
all of the *.mu in mmcv/ops/csrc/pytorch/musa are ported for  mmcv/ops/csrc/pytorch/cuda

**Comment by hanhaowen-mt** [comment]
Since non-py files are controlled by setup.py, there is no need to review non-py  files during review

**Comment by hanhaowen-mt** [comment]
emmmm好几个月了都没人来review....

**Comment by hanhaowen-mt** [comment]
@zhouzaida 哥们能不能来瞅一瞅。。。。

File: docs/zh_cn/mmcv-logo.png
Comment (zhouzaida) [inline docs/zh_cn/mmcv-logo.png:1]: Why is this file removed?

Comment (hanhaowen-mt) [reply docs/zh_cn/mmcv-logo.png:1]: 已恢复
